src/overlays/actors/ovl_Obj_Lightswitch/z_obj_lightswitch.h

```diff
@@ -6,9 +6,28 @@
 
 struct ObjLightswitch;
 
+typedef void (*ObjLightswitchActionFunc)(struct ObjLightswitch*, GlobalContext*);
+
+enum ObjLightswitch_Type {
```

Comment:
I guess I missed this one but the enum style should probably be adjusted here as well.